async-query/src/main/java/org/opensearch/sql/spark/config/OpenSearchSparkSubmitParameterModifier.java

```diff
@@ -6,15 +6,15 @@
 package org.opensearch.sql.spark.config;
 
 import lombok.AllArgsConstructor;
-import org.opensearch.sql.spark.asyncquery.model.SparkSubmitParameters;
+import org.opensearch.sql.spark.parameter.SparkSubmitParametersBuilder;
 
 @AllArgsConstructor
 public class OpenSearchSparkSubmitParameterModifier implements SparkSubmitParameterModifier {
 
   private String extraParameters;
 
   @Override
-  public void modifyParameters(SparkSubmitParameters parameters) {
-    parameters.setExtraParameters(this.extraParameters);
+  public void modifyParameters(SparkSubmitParametersBuilder builder) {
```

Comment:
could u explain more on Modifier and Composer and add to PR description. are these necessary? could we simply the concept?

Reply:
Added description. I am planning to delete Modifier since Generic composer will cover the use case.